src/parse/ast/node.rs

```diff
@@ -181,9 +183,10 @@ impl Node {
                 ty: Box::from(ty.map(mapping)),
                 args: args.iter().map(|a| a.map(mapping)).collect(),
             },
-            Node::Reassign { left, right } => Node::Reassign {
+            Node::Reassign { left, right, op } => Node::Reassign {
                 left: Box::from(left.map(mapping)),
                 right: Box::from(right.map(mapping)),
+                op, // op is untouched
```

Comment:
If we use an special `NodeOp` enum, then we don't need to explain ourselves in a comment here!